src/helm/benchmark/run_expander.py

```diff
@@ -291,6 +307,65 @@ def expand(self, run_spec: RunSpec) -> List[RunSpec]:
         ]
 
 
+class OpenAIRunExpander(RunExpander):
```

Comment:
I originally had OpenAI and Google run expanders because I had different prompts at some point...how about having a `InContextLearningInstructionsRunExpander` class and then just having OpenAI and Google inherit from it?  In the future, we might tweak things.

Reply:
Added a todo to deal with this later.